src/main/scala/org/openhorizon/exchangeapi/route/agent/AgentConfigurationManagement.scala

```diff
@@ -61,12 +61,8 @@ trait AgentConfigurationManagement extends JacksonSupport with AuthenticationSup
                   certificate <- AgentCertificateVersionsTQ.delete
                   
                   timestamp: Timestamp = ApiTime.nowUTCTimestamp
-                  
```

Comment:
This introduces a severe bug. `.update()` can only update existing records in the DB, it cannot insert new records. This change eliminates the insert record path. The warning thrown is likely a syntactical-sugar bug with Slick, can be safely ignored here. The runtime evaluation of the expressions is correct and the type equivalence check holds true, via Slick type lifting. If you are dead set on removing the warning the following would work:

```
                  someVariableA <-
                    AgentVersionsChangedTQ.getChanged("IBM").result

                  changed <-
                    if(someVariableA == 0)
                      AgentVersionsChangedTQ += (timestamp, "IBM")
                    else
                      AgentVersionsChangedTQ.getChanged("IBM").update(timestamp)
```

Reply:
Agree. Fixed. @naphelps Maybe it would be better to update the unit test to cover this case?